File: ex1.py
# Implemente um programa que solicite 10 números inteiros informados pelo usuário. Exiba a soma

soma = 0
for i in range(10):
    numero = int(input("Digite um número inteiro: "))

    # Adiciona o número a soma total
    soma += numero

print("A soma dos números é:", soma)

File: Java/ex1.java
// Implemente um programa que solicite 10 números inteiros informados pelo usuário. Exiba a soma.

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int soma = 0;

        for (int i = 0; i < 10; i++) {
            System.out.println("Digite um número inteiro:");
            int num = scanner.nextInt();
            soma += num;
        }

        System.out.println("A soma dos números é: " + soma);
    }
}
